### PR TITLE
Add Italian translation file

### PR DESCRIPTION
**Pull request by alsoknownasdrew**
Add Italian translation as requested in  #17

### CODE REVIEW

File: i18n/bot.it-it.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  startolduser: |
+    <b>Bentornato</b>!
+    I tuoi indirizzi e-mail registrati sono:
+    {} {}
+    {} {}
+    Per inviare il file al tuo Kindle, clicca <b>Invia file</b>.
+    Per modificare il tuo indirizzo e-mail, clicca <b>Modifica e-mail</b>
+  error: |
+    <b>Errore</b>.
+    Clicca /inizia e verifica i tuoi indirizzi e-mail.
```
Comment (GabrielRF) [inline i18n/bot.it-it.yml]: /start should not be translated

**Comment by alsoknownasdrew** [comment]
Fixed it

File: i18n/bot.it-it.yml (hunk 1 above)
Comment (alsoknownasdrew) [reply i18n/bot.it-it.yml]: Fixed :)

**Comment by GabrielRF** [comment]
@alsoknownasdrew I'm not sure which PR should be accepted
Can you help me at #17 ?

**Pull request closed without merging**